server/server.js

```diff
@@ -0,0 +1,49 @@
+var express=require('express');
+var path = require('path');
+var fs = require('fs');
+var ff = require('ff');
+var uuid = require('node-uuid');
+var mv = require('mv');
+
+const PUBLIC_PATH = '../../public_static/';
+const PORT = 3001;
+
+function upload(req, res) {
+  var name = req.query.name;
+  var description = req.query.description;
+  var metaContent = name + '\n' + description;
+
+  console.log('got one!', name, description);
+
+  var oldPath = req.headers['x-file'];
+  var newPath = path.resolve(__dirname, PUBLIC_PATH);
+
+  var prefix = name.replace(' ', '_') + '__' + uuid.v1();
```

Comment:
yeah, you're probably right. it was nice for debugging though. i'll just remove the prefix.